superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -426,43 +484,92 @@ int RunCopy(const BenchArgs &args, const Buffers &buffers) {
     }
 
     // Create stream to launch kernels
+    if (SetGpu(args.worker_gpu_id)) {
+        return -1;
+    }
     cuda_err = cudaStreamCreateWithFlags(&stream, cudaStreamNonBlocking);
     if (cuda_err != cudaSuccess) {
         fprintf(stderr, "RunCopy::cudaStreamCreate error: %d\n", cuda_err);
         return -1;
     }
+    // P2P bidirecitonal test, create a second stream
+    if (args.is_bidirectional) {
+        if (SetGpu(args.worker_gpu_id_2)) {
+            return -1;
+        }
```

Comment:
SetGpu before the cudaStreamCreateWithFlags

Reply:
SetGpu at line 487 is for cudaStreamCreateWithFlags at line 490, and SetGpu at line 497 is for cudaStreamCreateWithFlags at line 500.